data_scraper/validation.py

```diff
@@ -63,9 +67,12 @@ def validate_historical_dates(symbol, date_range):
 
     return missing_days.empty
 
+
 def validate_columns(expected, received):
     """Verify that the `received` columns scraped are equal to `expected`"""
-    valid = all(expected == received)
+    expected = sorted(expected)
+    received = sorted(received)
+    valid = expected == received
```

Comment:
This will not work, `expected` and `received` are list-like objects.  
To compare them for equality you should use `all()` like so:
```python
>>> valid = all(expected == received)
```